Updates to information about TrialNotAllowed (#2654)
GoogleChrome/developer.chrome.com: site/en/blog/origin-trial-troubleshooting/index.md

{% endAside %}

### The origin trial is available for the current user {: #user}

> Not all origin trials are available for all users, even if a valid token is provided.

Reviewer: ```suggestion
Some origin trials are unavailable to certain users, even if a valid token is provided.
```

Reviewer: Framing this in positive instead of negative language will make it easier to localize and understand.

If a trial isn't available for the current user, Chrome DevTools will display a `TrialNotAllowed` warning:

{% Img src="image/80mq7dk16vVEg8BBhsVe42n6zn82/jToK0McIe9AgqCS3rymo.png", alt="Chrome DevTools 
origin trials information in the Application panel showing TrialNotAllowed warning", width="800", 
height="424" %}

Information about usage restrictions and availability will be provided for each origin trial.

> As with any web platform feature, you should use [feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) to check support 
> for an origin trial feature before using it.

Reviewer: ```suggestion
As with any web platform feature, you should use [feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection)
to confirm that an origin trial feature is supported before you use it.
```
